### DISCUSSION TITLE
Inquiry about policy for incomplete pull requests

### DISCUSSION THREAD

**Discussion by unereal** [c3lang/c3c#1188] [General]
I'm opening this topic to inquire about C3C policy regarding incomplete Pull Requests. Currently, I've implemented a solution for Linux, but unfortunately, I don't have access to Windows and Mac platforms to perform corresponding tests and implementations. I would like to know if it's acceptable to submit partial Pull Requests in this scenario or if there are specific guidelines I should follow.

**Comment by lerno** [comment]
Yes, that is fine.
